test/e2e/proxy_test.go

```diff
@@ -253,6 +253,32 @@ func TestProxyFlow(t *testing.T) {
 				assert.Equal(t, expectedApp.Spec.DisplayName, createdApp.Spec.DisplayName)
 			}) // end of successful workspace context request
 
+			t.Run("successful workspace context request with proxy plugin", func(t *testing.T) {
+				// we are going to repurpose a well known, always running route as a proxy plugin to contact through the registration service
+				CreateProxyPluginWithCleanup(t, hostAwait, "openshift-console", "openshift-console", "console")
+				VerifyProxyPlugin(t, hostAwait, "openshift-console")
+				proxyPluginWorkspaceURL := hostAwait.PluginProxyURLWithWorkspaceContext("openshift-console", user.compliantUsername)
```

Comment:
Did you forget to commit `PluginProxyURLWithWorkspaceContext`?

Reply:
yes I did ... just pushed it